From bf87f952dc7a07786ddb9e895a956505cd951cf3 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Mon, 8 Aug 2016 10:12:33 -0700
Subject: [PATCH 32/32] Ticket bz1358565 -  clear and unsalted password types
 are vulnerable to timing attack

Description: Fixing a compiler warning introduced by commit
f0e03b5a51972a125fe78f448d1f68e288782d1e.
---
 ldap/servers/plugins/pwdstorage/clear_pwd.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ldap/servers/plugins/pwdstorage/clear_pwd.c b/ldap/servers/plugins/pwdstorage/clear_pwd.c
index 2afe16e..b9b362d 100644
--- a/ldap/servers/plugins/pwdstorage/clear_pwd.c
+++ b/ldap/servers/plugins/pwdstorage/clear_pwd.c
@@ -26,7 +26,6 @@ int
 clear_pw_cmp( const char *userpwd, const char *dbpwd )
 {
     int result = 0;
-    int len = 0;
     int len_user = strlen(userpwd);
     int len_dbp = strlen(dbpwd);
     if ( len_user != len_dbp ) {
-- 
2.4.11

